05-deploying_shiny_app.Rmd

```diff
@@ -1,279 +1,239 @@
-
-![](images/shiny/use_ssh.PNG)
-
-![](images/shiny/ssh_url.PNG)
+### Manage dependencies
 
+Most apps will have dependencies on various third-party packages. Changes to these packages may not always be backwards-compatible. To avoid compatibility issues and ensure reproducible outputs, it is necessary to use a package management system, such as packrat or conda.
 
-### Work on your app
+If using packrat, ensure that it is enabled for your project in RStudio.
 
-You can now work on your Shiny app in R Studio as normal.  As you work, commit your changes to Github using your chosen Github workflow.
+To enable packrat, select 'Tools' > 'Project Options...' > ' Packrat' > 'Use packrat with this project'.
 
-### Snapshot dependencies using packrat
+When packrat is enabled, run `packrat::snapshot()` to generate a list of packages used in the project, their sources and their current versions.
 
-Your Shiny app will have dependencies on various third-party R packages (e.g. `dplyr`).  These packages change through time, and sometimes these changes are not backwards compatible.  When we deploy your Shiny app, it is therefore essential to include a manifest of all of the package dependencies and their specific versions.
+You may also wish to run `packrat::clean()` to remove unused packages from the list.
 
-The R package`packrat` automates this process.  Before deploying your app, you need to call `packrat::snapshot()` to automatically create this manifest.  You may want to call `packrat::clean()` to remove any dependencies that are no longer in use.
+The list is stored in a file called `packrat/packrat.lock`. You must ensure that you have committed this file to GitHub before deploying your app.
 
-This creates a file called `packrat/packrat.lock`.  You need to ensure you have committed this to the Github repository prior to deployment.
+### Set access permissions
 
-![](images/shiny/github_packrat_snapshot.gif)
+You can set some access permissions for your app in the `deploy.json` file.
 
-### Build and deploy
+The `allowed_ip_ranges` parameter controls where your app can be accessed from. It can take any combination of the following values `["DOM1", "QUANTUM", "102PF WiFi"]` or `["Any"]`.
 
-Include a `deploy.json` file so that Concourse will automatically build and deploy your app, so that it is running and customers can access it. For more about this see: [Build and deploy guidance](https://moj-analytical-services.github.io/platform_user_guidance/build-and-deploy.html)
+The `disable_authenication` parameter controls whether sign-in is required for users to access the app. It can take the values `true` or `false`. In general, this should be set to `false`. If you wish to disable authentication, you should first discuss this with the Analytical Platform team.
 
-### Grant secure access to the app{#grant-app-access}
+### Create a release in GitHub
 
```

Comment:
How about explaining why you're doing this step. It's not obvious. Something like: `When you’re ready to share the app, in GitHub create a ‘release’ and it will deploy automatically:`

Reply:
Further explanation in d2fef17.